.circleci/config.yml

```diff
@@ -20,12 +20,6 @@ jobs:
           command: |
```

Comment:
I can't add a comment on line 16, but you can remove the pip and setuptools pinning since it's not required for python 3.8

Reply:
Will do